sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/ExpressionParserSuite.scala

```diff
@@ -433,10 +433,13 @@ class ExpressionParserSuite extends AnalysisTest {
     intercept("timestamP '2016-33-11 20:54:00.000'", "Cannot parse the TIMESTAMP value")
 
     // Interval.
-    val intervalLiteral = Literal(IntervalUtils.fromString("interval 3 month 1 hour"))
+    var intervalLiteral = Literal(IntervalUtils.fromString("interval 3 month 1 hour"))
     assertEqual("InterVal 'interval 3 month 1 hour'", intervalLiteral)
     assertEqual("INTERVAL '3 month 1 hour'", intervalLiteral)
     intercept("Interval 'interval 3 monthsss 1 hoursss'", "Cannot parse the INTERVAL value")
+    assertEqual("+interval '3 month 1 hour'", intervalLiteral)
+    intervalLiteral = Literal(IntervalUtils.fromString("interval -3 month -1 hour"))
+    assertEqual("-interval '3 month 1 hour'", intervalLiteral)
```

Comment:
If you don't mind, let's use `val intervalLiteral2` instead of touching line 436.
```scala
-    intervalLiteral = Literal(IntervalUtils.fromString("interval -3 month -1 hour"))
-    assertEqual("-interval '3 month 1 hour'", intervalLiteral)
+    val intervalLiteral2 = Literal(IntervalUtils.fromString("interval -3 month -1 hour"))
+    assertEqual("-interval '3 month 1 hour'", intervalLiteral2)
```